### PR TITLE
#16 actor selection

### PR DESCRIPTION
**Pull request by mjanczykowski**
Not tested yet on Prometheus due to some serious malfunction.

### CODE REVIEW

File: common/src/main/scala/miss/trafficsimulation/actors/AreaActor.scala [hunk 1]
```diff
@@ -19,11 +20,16 @@ class AreaActor(config: Config) extends FSM[State, Data] {
 
   startWith(Initialized, EmptyData)
 
+  private def resolveActor(actorPath: ActorPath): ActorSelection = this.context.actorSelection(actorPath)
```
Comment (salceson) [inline common/src/main/scala/miss/trafficsimulation/actors/AreaActor.scala]: I don't like this method name.

Comment (salceson) [reply common/src/main/scala/miss/trafficsimulation/actors/AreaActor.scala]: What is more - is `this` needed here?

**Review by salceson — CHANGES_REQUESTED** [review]

**Comment by salceson** [comment]
Is this necessary?

**Comment by mjanczykowski** [comment]
I hope not. I'll keep this branch locally and close this PR.

**Pull request closed without merging**